### DISCUSSION TITLE
How can I use fuzzing, When Payload was changing based on selected example

### DISCUSSION THREAD

**Discussion by AkhilVeeramreddi-CS** [apideck-libraries/portman#511] [Q&A]
Hi there,

I'm facing an issue when I'm using fuzzing. Tests are generated by referring to OAS specifications. As per our spec payloads were changed based on examples from dropdown. 
 ![Screenshot 2023-09-11 140007](https://github.com/apideck-libraries/portman/assets/126651746/cfb2d0d0-26fe-4231-99e8-8de285bc4d27)

![MicrosoftTeams-image (4)](https://github.com/apideck-libraries/portman/assets/126651746/cb71efa9-5fb2-4bee-a07e-5745dab423ac)
The above two attachments are the requirements.

Following script was how I write the script.
![image](https://github.com/apideck-libraries/portman/assets/126651746/d6d9769d-bc3e-4297-9e32-3d0119a387d9)

I hope that this provides some context.

Kind regards,
Akhil

**Comment by thim81** [comment]
hi @AkhilVeeramreddi-CS 

It is not clear what you are expecting. Could you share more info, so we can think about possible options?
